crates/napi/src/next_api/project.rs

```diff
@@ -988,7 +989,14 @@ async fn output_assets_operation(
         .flat_map(|assets| assets.iter().copied())
         .collect();
 
-    Ok(Vc::cell(output_assets.into_iter().collect()))
+    let nft = next_server_nft_assets(container.project()).await?;
+
+    Ok(Vc::cell(
+        output_assets
+            .into_iter()
+            .chain(nft.iter().copied())
+            .collect(),
+    ))
```

Comment:
In case of bugs with this, should we provide a way (an `experimental` flag?) to opt out of the Rust-based NFT implementation from the `next.config.js` file?

Reply:
I'd argue that this PR is actually the less dangerous of the NFT changes.
The per-route NFT files (that handle user code) are more prone to breaking (as we are seeing in these days). This is only tracing Next internal code, which is working fine and doesn't really differ across apps.